app/src/main/java/com/frinder/frinder/activity/LoginActivity.java

```diff
@@ -78,7 +81,7 @@ public void onCompleted(JSONObject object, GraphResponse response) {
                             }
                         });
                 Bundle parameters = new Bundle();
-                parameters.putString("fields", "id,name,email,gender");
+                parameters.putString("fields", "id,name,email,gender,age_range,link");
```

Comment:
style nit: inconsistent spacing. spaces between ','

Reply:
For style issues, formatting using Android Studio should help us be consistent. I think Cmf +Opt +L does that.